test/close.test.js

```diff
@@ -77,6 +77,29 @@ test('close order', t => {
   })
 })
 
+test('close order - async', async t => {
+  t.plan(3)
+  const fastify = Fastify()
+  const order = [1, 2, 3]
+
+  fastify.register(function (f, opts, next) {
+    f.addHook('onClose', async instance => {
+      t.is(order.shift(), 1)
+    })
+
+    next()
+  })
+
+  fastify.addHook('onClose', () => {
+    t.is(order.shift(), 2)
+  })
+
+  await fastify.listen(0)
+  await fastify.close(0)
```

Comment:
why 0?

Reply:
Because it was a bad copy/paste. :) Removing it.

Reply:
Removed.